docs/scheduled-transactions/bdds.md

```diff
@@ -0,0 +1,105 @@
+
+Expect:
+- New Schedule entity must be created and its ID must be specified in the `TransactionReceipt`'s `ScheduleID` property.
+- When querying the Schedule Entity by the `ScheduleID` form the `transactionReceipt`:
+    - it must have the same `ScheduleID` as specified in the `TransactionReceipt`
+    - it must have `creatorAccountID` set, equal to the `transaction sender` of the `schedule create` transaction
+    - **it must have `payerAccountID` set to the `payerAccountID` specified in the creation TX**
+    - it must have the same `transactionBody` bytes retrieved
+    - it must have empty `adminKey`
+    - it must have empty `signers` 
+    - it must have empty `memo`
+    
+**4. Should be able to create Scheduled Transaction with `transactionBody` and `memo`**
+
+Expect:
+- New Schedule entity must be created and its ID must be specified in the `TransactionReceipt`'s `ScheduleID` property.
+- When querying the Schedule Entity by the `ScheduleID` form the `transactionReceipt`:
+    - it must have the same `ScheduleID` as specified in the `TransactionReceipt`
+    - it must have `creatorAccountID` set, equal to the `transaction sender` of the `schedule create` transaction
+    - it must have `payerAccountID` set, equal to the `transaction sender` of the `schedule create` transaction
+    - it must have the same `transactionBody` bytes retrieved
+    - it must have empty `signers` 
+    - it must have empty `adminKey`
+    - **it must have `memo` set, equal to the `memo` specified in the creation TX**
+
+**5. Should be able to create Scheduled Transaction with `transactionBody` and `signatures`**
+
+When creating the Scheduled Transaction, populate the `SignatureMap` with one or more signatures by `Key`(s) required by the transaction that is being scheduled.
+
+Expect: 
+- New Schedule entity must be created and its ID must be specified in the `TransactionReceipt`'s `ScheduleID` property.
+- When querying the Schedule Entity by the `ScheduleID` form the `transactionReceipt`:
+    - it must have the same `ScheduleID` as specified in the `TransactionReceipt`
+    - it must have `creatorAccountID` set, equal to the `transaction sender` of the `schedule create` transaction
+    - it must have `payerAccountID` set, equal to the `transaction sender` of the `schedule create` transaction
+    - it must have the same `transactionBody` bytes retrieved
+    - **it must have a set of `signers` populated. The signers must correspond to the simple `Key`(s) whose signature has been provided.** 
+    - it must have empty `adminKey`
+    - it must have empty `memo` 
+    
+    
+- create scheduled tx, create second one (only with same tx bytes) and check that new one was created
+- create scheduled tx, create second one (with same tx bytes, admin) and check that new one was created
+- create scheduled tx, create second one (with same tx bytes, admin and payer) and check that sigs were appended
```

Comment:
create scheduled tx, create second one (with same tx bytes, payer) and check that new one was created